repose-aggregator/components/filters/uri-normalization-filter/build.gradle

```diff
@@ -1,8 +1,10 @@
 dependencies {
     compile project(":repose-aggregator/commons/commons-configuration")
+    compile project(":repose-aggregator/core/core-lib")
```

Comment:
Not actually, though.

We have a bunch of classes in core-lib for reporting metrics. In this case, the `UriNormalization` class is in core-lib, and is used in this filter for reporting metrics.

It's... kind of dumb.